packages/clay-language-picker/src/index.tsx

```diff
@@ -0,0 +1,285 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {Option, Picker} from '@clayui/core';
+import ClayIcon from '@clayui/icon';
+import ClayLabel from '@clayui/label';
+import ClayLayout from '@clayui/layout';
+import {InternalDispatch, sub} from '@clayui/shared';
+import classNames from 'classnames';
+import React from 'react';
+
+type DisplayType = 'info' | 'secondary' | 'success' | 'warning';
+
+type Item = {
+	displayName?: string;
+	href?: string;
+	id: string;
+	label: string;
+	symbol: string;
```

Comment:
Looking at the examples and other cases it seems that we will almost always have the same data for this API, I would say that we can only keep one which would be the `symbol` prop and it could even be `id` as a name. We can use this value as an identifier for both the icons and for the rest of the other APIs, such as selectedLocale... Apparently these other values ​​would be unnecessary for us here.

Reply:
`symbol` refers to the icon that will be displayed and `label` is the label that we want to be displayed in the picker option.  In my opinion these two props are independent of the `id` prop. I think it is important to maintain and differentiate these keys. Maybe we want to show other icon or other label, and that doesn't have to affect the prop `id`. What do you think?

Reply:
Hmm okay, we can go with that now, I know there are some cases of the same symbols for some flags but if I'm mistaken we repeat the icon but keep the symbol name different.